maven/impl-maven-embedded/src/test/java/org/jboss/shrinkwrap/resolver/impl/maven/embedded/ConfigurationStageTestCase.java

```diff
@@ -32,7 +34,7 @@ public class ConfigurationStageTestCase {
         put("propertyKey1", "propertyValue1");
         put("propertyKey2", "propertyValue2");
     }};
-    final Map<String, String> shellEnvironments = new HashMap() {{
+    final Map<String, String> shellEnvironments = new HashMap<String, String>() {{
```

Comment:
```suggestion
    final Map<String, String> shellEnvironments = new HashMap<>() {{
```

Reply:
This is actually invalid due to the anonymous inner class, `new HashMap<>()` won't even compile

This would work on JDK 9+, we're still on JDK 8